test/mysql.test.js

```diff
@@ -155,7 +155,73 @@ describe('mysql', function () {
     });
 
   });
+  
```

Comment:
context('replaceOrCreate', function() {
  it('should replace the instance', function(done) {
     ...
  });
  ...
});